File: data/src/main/java/mega/privacy/android/data/model/ChatSettings.java
package mega.privacy.android.data.model;

public class ChatSettings {

    private String notificationsSound;
    private String vibrationEnabled;
    private String videoQuality;

    public ChatSettings(String notificationsSound, String vibrationEnabled, String videoQuality) {
        this.notificationsSound = notificationsSound;
        this.vibrationEnabled = vibrationEnabled;
        this.videoQuality = videoQuality;
    }

    public ChatSettings() {
        this.notificationsSound = "";
        this.vibrationEnabled = true+"";
        this.videoQuality = false+"";
    }

    public String getNotificationsSound() {
        return notificationsSound;
    }

    public void setNotificationsSound(String notificationsSound) {
        this.notificationsSound = notificationsSound;
    }

    public String getVibrationEnabled() {
        return vibrationEnabled;
    }

    public void setVibrationEnabled(String vibrationEnabled) {
        this.vibrationEnabled = vibrationEnabled;
    }

    public String getVideoQuality() {
        return videoQuality;
    }

    public void setVideoQuality(String videoQuality) {
        this.videoQuality = videoQuality;
    }
}

File: domain/src/main/kotlin/mega/privacy/android/domain/entity/settings/ChatSettings.kt
package mega.privacy.android.domain.entity.settings

import mega.privacy.android.domain.entity.VideoQuality

/**
 * Data class for storing chat settings.
 *
 * @property notificationsSound
 * @property vibrationEnabled
 * @property videoQuality
 */
data class ChatSettings(
    val notificationsSound: String = "",
    val vibrationEnabled: String = VIBRATION_ON,
    val videoQuality: String = VideoQuality.MEDIUM.value.toString(),
) {
    companion object {
        /**
         * Notification vibration on.
         */
        const val VIBRATION_ON = true.toString()

        /**
         * Notification vibration off.
         */
        const val VIBRATION_OFF = false.toString()
    }
}